plugins/modules/redfish_storage_volume.py

```diff
@@ -583,6 +641,15 @@ def validate_inputs(module):
                          " volume_id must be specified to perform further actions.")
 
 
+def map_volume_type(volume_type):
+    volume_type_map = {"NonRedundant": "RAID0",
```

Comment:
Can this be a constant, and just do map.get(volume_type) to get instead of a method.